packages/express/lib/serve.js

```diff
@@ -1,29 +1,31 @@
+const debug = require('debug')('untool:express');
 const express = require('express');
-const helmet = require('helmet');
-const mime = require('mime');
 
-const rewriteMiddleware = require('./rewrite');
-
-module.exports = (method, options, config, initialize, finalize) => {
-  const app = express();
-  initialize(app, method);
-  app.use(rewriteMiddleware(options, config));
-  app.use(helmet());
-  app.use(
-    express.static(config.buildDir, {
-      maxAge: '1y',
-      setHeaders: function(res, filePath) {
-        if (
-          (res && res.locals && res.locals.noCache) ||
-          mime.getType(filePath) === 'text/html'
-        ) {
-          helmet.noCache()(null, res, function() {});
-        }
-      },
-      redirect: false,
-    })
+module.exports = (mode, { configureServer }) => {
+  const phases = ['initial', 'files', 'parse', 'routes', 'final'];
+  const middlewares = phases.reduce(
+    (result, key) => {
+      const additions = { [`pre${key}`]: [], [key]: [], [`post${key}`]: [] };
+      return {
+        ...result,
+        ...additions,
+        middlewareOrder: result.middlewareOrder.concat(Object.keys(additions)),
+      };
+    },
+    { middlewareOrder: [] }
+  );
+  const app = configureServer(express(), middlewares, mode);
+  const { middlewareOrder } = middlewares;
+  debug(middlewares);
```

Comment:
It would be great if we could print the middlewares in the order they will be used.

Reply:
`middlewareOrder` is being already being printed, as it is a property of `middlewares` - and, in my opinion, it is quite useful to at least have the number of middlewares defined for each phase.